src/api/operations/enrollment.fragments.graphql

```diff
@@ -260,3 +260,17 @@ fragment EnrollmentSummaryFields on EnrollmentSummary {
   projectType
   canViewEnrollment
 }
+
+fragment EnrollmentAuditEventFields on EnrollmentAuditEvent {
```

Comment:
not sure that we need both EnrollmentAuditEvent and ClientAuditEvent; feels like it could be simplified